### DISCUSSION TITLE
Overriding YimMenu's No Recoil/No Sway

### DISCUSSION THREAD

**Discussion by MindOfBog** [YimMenu/YimMenu#3646] [Q&A] [answered]
I need to be able to disable recoil/sway for a mod I'm making, however I can't figure out how, I have tried out the command for them and that works fine, however I cannot read the current state of those booleans to then revert them after so that the users settings aren't being changed.

Is there a way to override them completely without affecting the users settings? Or maybe some way of reading the checkboxes states so that I can revert the settings after?

If there is no way, I can always just add a note so the user changes those settings themselves..

**Comment by xesdoog** [comment, marked as answer]
Unfortunately there is no way to check which options are enabled as you'd need access to the `settings.json` file and YimMenu's Lua is limited to only the *scripts* and *scripts_config* folders.
